server/package.json

```
@@ -5,9 +5,9 @@
   "description": "The backend for Ask.gov.sg",
   "main": "./build/index.js",
   "scripts": {
-    "start": "node ./build/index.js",
+    "start": "node -r tsconfig-paths/register ./build/index.js",
     "build": "tsc -b",
-    "dev": "ts-node-dev --respawn --transpile-only --inspect -- ./src/index.ts",
+    "dev": "ts-node-dev -r tsconfig-paths/register --respawn --transpile-only --inspect -- ./src/index.ts",
```

tried it out, the TS recompiles but the server doesn't restart when changes are made to `shared`. easy fix:
```suggestion
    "dev": "ts-node-dev -r tsconfig-paths/register --respawn --transpile-only --inspect --watch ../shared/src -- ./src/index.ts",
```